### ISSUE TITLE
Issue: Cannot locate MSBuild engine for .NET SDK v9.0.100

### ISSUE THREAD

**Issue by jrdodds** [tintoy/msbuild-project-tools-vscode#157]
With Visual Studio 2022 Preview installed, the .NET SDK 9.0.100-preview is installed and MSBuild Project Tools for Visual Studio Code fails with the following output:

```
System.Exception: Cannot locate MSBuild engine for .NET SDK v9.0.100. This probably means that MSBuild Project Tools cannot find the MSBuild for the current project instance. It did find the following version(s), though: [8.0.400, 8.0.304, 7.0.410, 6.0.425].
   at MSBuildProjectTools.LanguageServer.Utilities.MSBuildHelper.DiscoverMSBuildEngine(String baseDirectory, ILogger logger)
   at MSBuildProjectTools.LanguageServer.Program.Main()
[Error - 10:44:20 AM] Server process exited with code 1.
[Error - 10:44:20 AM] Connection to server got closed. Server will not be restarted.
[Error - 10:44:20 AM] MSBuild Language Service client: couldn't create connection to server.
  Message: Pending response rejected since connection got disposed
  Code: -32097 
```

The command `dotnet --list-sdks` reports:

```
6.0.425 [C:\Program Files\dotnet\sdk]
7.0.410 [C:\Program Files\dotnet\sdk]
8.0.304 [C:\Program Files\dotnet\sdk]
8.0.400 [C:\Program Files\dotnet\sdk]
9.0.100-preview.7.24407.12 [C:\Program Files\dotnet\sdk]
```

**Comment by tintoy** [comment]
Hi , thanks for reporting!

I suspect that this may be the same issue as #144 . While it was partially addressed by #152,  we still don’t have great support for preview versions but our next release has some additional improvements that may help (using preview runtimes / SDKs is always a bit of an unknown quantity for us since it depends whether the new ones are backward-compatible with the one we built against).

I’m hoping to put our new release together this weekend 🙂

CC: @DoctorKrolic

**Comment by jrdodds** [comment]
Yes, it seems to be related to #144. Thanks

**Comment by jrdodds** [comment]
Resolved by #158 (Release v0.6.5)

Output for "MSBuild Project Tools" reports:

```
Starting MSBuild language service...
Using .NET runtime from the host
MSBuild language service is running.
```

And output for "MSBuild language service" reports "`Using MSBuild engine v9.0.100 from "C:\Program Files\dotnet\sdk\9.0.100-preview.7.24407.12".`"

**Comment by jrdodds** [comment]
Resolved by #158 (Release v0.6.5)

**Comment by tintoy** [comment]
Thanks!
